README.md

```diff
@@ -55,27 +57,32 @@ cd functions
 # Install the function dependencies
 npm install
 
+# Go back to the root app directory to use firebase package
+cd ..
+
 # Login to firebase with the account that has access to this project
-firebase login
+npm run firebase login
 
 # Switch to the 'staging' project in our local configuration (or whichever environment you are developing against)
-firebase use staging
+npm run firebase use staging
 
 # Copy the runtime config locally
-npm run firebase functions:config:get > .runtimeconfig.json
+npm run --silent firebase functions:config:get > .runtimeconfig.json
 ```
 
 Now you're ready to launch the backend function emulator! ✨
 
 ```bash
+# Run the command in the root app directory
 npm run firebase:emulate-functions
 
 # Or if you don't want to use our helper scripts, you can do this directly:
-# firebase emulators:start --only functions
+# npm run firebase emulators:start --only functions
 ```
 
 **Note**: You might need to emulate the firebase functions locally before the server can properly start. If you have issues using/editing the actual staging functions, try that.
 
+**Note**: If you're interested in setting up the project to run in your own environment but you're still unclear about how to do that feel free to open an issue asking for assistance.
```

Comment:
Let's move this right up the top directly under `## Getting started`

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1303/commits/57b9fa921e25b350ee301bd6b068526b2d3dcfd4